projects/GKCore/GKCore/LangMan.cs

```diff
@@ -1673,7 +1673,7 @@ public static class LangMan
             /* 781 */ "Adopted",
             /* 782 */ "Birth",
             /* 783 */ "Foster",
-            /* 784 */ "Charset detection at load",
+            /* 784 */ "Detect Charset at load",

```

Comment:
This is checkbox in options. Imperative mood sounds more appropriate here.

Reply:
👍